libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -151,12 +163,14 @@ export class SkyActionButtonContainerComponent
   }
 
   private updateResponsiveClass(): void {
```

Comment:
Private methods should also use the hash privates

Reply:
done